scarb/src/compiler/compilers/lib.rs

```diff
@@ -60,11 +61,15 @@ impl Compiler for LibCompiler {
 
         if props.sierra {
```

Comment:
We need to give users an ability to generate textual representation, in case some tooling relies on it. Let's do that in separate task, that should be quite high priority, becase we need to deliver it in the same release as this PR: #634